.github/workflows/deploy-release.yml

```
@@ -151,6 +152,28 @@ jobs:
             ${{ steps.get-name.outputs.full-name }}.tar.gz
             ${{ steps.get-name.outputs.full-name }}.conda-lock.yml
 
+  create-git-tag:
+    name: Create Git Tag
+    runs-on: ubuntu-latest
+    needs: [update-dependencies, set-version]
+    # Run independently from the event if none of the needed jobs failed
+    if: ${{ !cancelled() && needs.update-dependencies.result == 'success' && needs.set-version.result == 'success' }}
+    env:
+      VERSION: ${{ needs.set-version.outputs.version }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          ref: main
+          token: ${{ secrets.REPO_CONTENT_WRITE_TOKEN }}
+          fetch-tags: true
+
+      - name: Create and push tag
+        shell: bash -el {0}
+        run: |
+            git tag ${{env.VERSION}}
```

You should annotate your tags with a message, even if it is something like: 
```suggestion
            git tag ${{env.VERSION}} -m "Tagged as part of ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```

---

Wow didn't know that GitHub context didn't have something like `run_url`

---

Yep, it annoys me every time I have to write out that string :D I usually put it in an `env` but still...

---

Fixed in [857f983b9206c20f187a9058444f54477b8f6807](https://github.com/ACCESS-NRI/access-ram-condaenv/pull/13/commits/857f983b9206c20f187a9058444f54477b8f6807)